src/MeadeCommandProcessor.cpp

```diff
@@ -1797,9 +1803,10 @@ String MeadeCommandProcessor::handleMeadeExtraCommands(String inCmd)
             long num = inCmd[2] - '0';
             if (num >= 0 && num < 2)
```

Comment:
This should be `if (num >=0 && num<= 9)` now if you want to support more than 2. But then `setHeater()` code will currently crash for `num>1` since `_heaters` is defined with size 2. So it seems you are only supporting two, so the doc should reflect that.

Reply:
See above.